db/fixtures/ae_datastore/ManageIQ/ConfigurationManagement/AnsibleTower/Service/Provisioning/StateMachines/Provision.class/catalogiteminitization.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
@bzwei 
Should the instance be named catalog_item_initialization instead of catalogiteminitization